libcoap/src/context.rs

```diff
@@ -646,6 +646,7 @@ impl Drop for CoapContextInner<'_> {
         // [as_mut_context()] contracts (we check validity of the pointer on construction).
         // Passing a NULL handler/None to coap_register_event_handler() is allowed as per the
         // documentation.
+        let a = 1.clone();
```

Comment:
Just as a reminder, to make sure, we should remember to remove these before merging.